karyon3-archaius2/src/main/java/com/netflix/karyon/archaius/ArchaiusKaryonSuite.java

```diff
@@ -0,0 +1,229 @@
+package com.netflix.karyon.archaius;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.multibindings.MapBinder;
+import com.google.inject.multibindings.Multibinder;
+import com.google.inject.name.Names;
+import com.netflix.archaius.Config;
+import com.netflix.archaius.config.CompositeConfig;
+import com.netflix.archaius.config.DefaultSettableConfig;
+import com.netflix.archaius.config.EnvironmentConfig;
+import com.netflix.archaius.config.MapConfig;
+import com.netflix.archaius.config.SettableConfig;
+import com.netflix.archaius.config.SystemConfig;
+import com.netflix.archaius.exceptions.ConfigException;
+import com.netflix.archaius.guice.ArchaiusModule;
+import com.netflix.archaius.guice.ConfigSeeder;
+import com.netflix.archaius.guice.ConfigSeeders;
+import com.netflix.archaius.guice.RootLayer;
+import com.netflix.archaius.inject.ApplicationLayer;
+import com.netflix.archaius.inject.DefaultsLayer;
+import com.netflix.archaius.inject.LibrariesLayer;
+import com.netflix.archaius.inject.RemoteLayer;
+import com.netflix.archaius.inject.RuntimeLayer;
+import com.netflix.governator.auto.AbstractPropertySource;
+import com.netflix.governator.auto.PropertySource;
+import com.netflix.karyon.AbstractKaryonSuite;
+import com.netflix.karyon.KaryonSuite;
+import com.netflix.karyon.ServerContext;
+
+/**
+ * Suite used to configure Archaius2 using a builder and making it available
+ * as a Governator PropertySource for conditional module loading as well as
+ * enabling it in the application.
+ * 
+ * @author elandau
+ *
+ */
+public abstract class ArchaiusKaryonSuite extends AbstractKaryonSuite {
```

Comment:
Why change name from Configuration to Suite? Is it because you're extending from another Suite? I thought Configuration tells exactly what this class does.